content/articles/vanity-nameservers.markdown

```diff
@@ -19,7 +19,7 @@ If you want to use our name servers but have them appear as though they were you
 
 ## Enabling Vanity Name Servers
 
-You can configure vanity name servers on your domain(s) in DNSimple by visiting the <label>DNS</label> tab on the domain's management page and clicking <label>Configure nameservers</label> on the <label>Vanity nameservers</label> card.
+You can configure vanity name servers on your domain(s) in DNSimple by visiting the <label>DNS</label> tab on the domain's management page and clicking <label>Configure name servers</label> on the <label>Vanity name servers</label> card.
```

Comment:
This change must be accompanied by a change in the UI, otherwise we leave the two out of sync.

/cc @sbastn @dallasread

Reply:
I have a PR up that corrects the spelling of name server across all of our pages. However, the correct action here would be `<label>Enable</label>` instead of `<label>Configure name servers</label>`